omod/src/main/java/org/openmrs/module/metadatasharing/web/controller/PublishController.java

```diff
@@ -226,7 +227,18 @@ public void getNewPackage(@RequestParam(required = false)
 								}
 							}
 						}
-					}
+						if(uuids==null && ids==null && modifiedSince!=null) {
```

Comment:
@reagan-meant  , wont you also need to check whether **type** is not empty ??

Reply:
@mozzy11 type has to always be specified...it isn't optional

Reply:
I think we actually can allow modifiedSince to be null... in which case it would fetch *all* objects of a certain type.